packages/arb-token-bridge-ui/src/components/TransferPanel/TokenButton.tsx

```diff
@@ -64,53 +65,57 @@ export function TokenButton(): JSX.Element {
 
   return (
     <>
-      <Popover className="h-full">
-        <Popover.Button
-          className="arb-hover h-full w-max rounded-bl-xl rounded-tl-xl bg-white px-3 hover:bg-gray-2"
-          aria-label="Select Token"
-        >
-          <div className="flex items-center space-x-2">
-            {tokenLogo && (
-              // SafeImage is used for token logo, we don't know at buildtime where those images will be loaded from
-              // It would throw error if it's loaded from external domains
-              // eslint-disable-next-line @next/next/no-img-element
-              <img
-                src={tokenLogo}
-                alt="Token logo"
-                className="h-5 w-5 sm:h-8 sm:w-8"
-              />
-            )}
-            <span className="text-xl font-light sm:text-3xl">
-              {tokenSymbol}
-            </span>
-            <ChevronDownIcon className="h-4 w-4 text-gray-6" />
-          </div>
-        </Popover.Button>
-        <Popover.Panel
-          className={twMerge(
-            panelWrapperClassnames,
-            'lg:ml-12 lg:min-w-[466px]'
-          )}
-        >
-          {({ close }) => {
-            function onClose() {
-              onPopoverClose()
-              close()
-            }
-            return (
-              <>
-                <div className="flex items-center justify-end border-b border-b-black px-5 py-4 lg:hidden">
-                  <button onClick={onClose}>
-                    <XMarkIcon className="h-8 w-8" />
-                  </button>
-                </div>
-                <div className="px-5 py-4">
-                  <TokenSearch close={onClose} />
-                </div>
-              </>
-            )
-          }}
-        </Popover.Panel>
+      <Popover className="relative">
+        {({ open }) => (
+          <>
+            <Popover.Button
+              className="arb-hover h-full w-max rounded-bl rounded-tl px-3 py-3 text-white"
+              aria-label="Select Token"
+            >
+              <div className="flex items-center gap-2">
+                {/* Commenting it out until we update the token image source files to be of better quality */}
```

Comment:
note that we are unsatisfied with the quality of the current token image files so we are hiding it for now on the button